src/VirtoCommerce.Platform.Caching/Redis/RedisPlatformMemoryCache.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Polly;
+using Polly.Retry;
+using StackExchange.Redis;
+using StackExchange.Redis.Extensions.Core;
+using VirtoCommerce.Platform.Caching;
+using VirtoCommerce.Platform.Core.Caching;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.Platform.Redis
+{
+    public class RedisPlatformMemoryCache : PlatformMemoryCache, IPlatformMemoryCache
+    {
+        private readonly ISubscriber _bus;
+        private readonly RedisCachingOptions _redisCachingOptions;
+        private readonly ISerializer _serializer;
```

Comment:
Could we use `JsonConverter.Serialize`  instead of injecting the  `ISerializer` as dependency?